docker-compose.yaml

```diff
@@ -29,12 +29,15 @@ services:
   create_chain_spec:
     container_name: create_chain_spec
     image: cerebellumnetwork/pos-node:2.1.0-rc1
+    user: root
```

Comment:
Why `root`?

Reply:
Because without these permissions the account generation does not work

Reply:
Also it is a one-time operation similar to how sudo is used to access a folder at one time.  

It is safer than changing permissions on all directories.

And these containers don't work all the time but only run once.  Boot_node and validators does not start from root